db/migrate/20190517093516_create_firmware_binary_firmware_constraints.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFirmwareBinaryFirmwareConstraints < ActiveRecord::Migration[5.0]
+  def change
+    create_table :firmware_binaries_firmware_constraints do |t|
+      t.bigint   :firmware_binary_id
+      t.bigint   :firmware_constraint_id
```

Comment:
Same, you probably don't want to `:index => true` on this but add a standalone `t.index %i[firmware_binary_id firmware_constraint_id], :unique => true` so there aren't extra indexes.

Reply:
Thanks. Turns out `:index => true` is default so I had to manually disable it.